apps/api/src/app/workflows-v2/usecases/generate-preview/generate-preview.usecase.ts

```diff
@@ -86,7 +86,11 @@ export class GeneratePreviewUsecase {
         const processedControlValues = this.fixControlValueInvalidVariables(controlValue, variables.invalidVariables);
 
         const validVariableNames = variables.validVariables.map((variable) => variable.name);
-        const variablesExampleResult = keysToObject(validVariableNames, { fn: (key) => `{{${key}}}` });
+        const showIfVariables: string[] = this.findShowIfVariables(processedControlValues);
+
+        const variablesExampleResult = keysToObject(validVariableNames, {
```

Comment:
Please add a comment about the need to override show variables with try for easier preview.

Reply:
done